File: find_diag_paths.php
<?php
$json = file_get_contents('/home/baga/Code/admixcenter/openapi.json');
$data = json_decode($json, true);

if (json_last_error() !== JSON_ERROR_NONE) {
    echo "JSON Error: " . json_last_error_msg() . "\n";
    exit;
}

if (!isset($data['paths'])) {
    echo "No paths found\n";
    exit;
}

foreach ($data['paths'] as $path => $details) {
    if (strpos($path, '/diagnostics') !== false || strpos($path, 'ping') !== false) {
        echo $path . "\n";
    }
}

File: find_diag_paths.py
import json

try:
    with open('/home/baga/Code/admixcentral/openapi.json', 'r') as f:
        data = json.load(f)
        paths = data.get('paths', {}).keys()
        diag_paths = [p for p in paths if '/diagnostics' in p]
        print("Diagnostics Paths:")
        for p in diag_paths:
            print(p)
except Exception as e:
    print(f"Error: {e}")
